tensorboard/webapp/metrics/views/main_view/card_grid_component.ts

```diff
@@ -46,14 +48,45 @@ export class CardGridComponent {
   @Output() pageIndexChanged = new EventEmitter<number>();
   @Output() groupExpansionToggled = new EventEmitter<void>();
 
-  showExpand(isBottomControl: boolean) {
+  constructor(
+    @Optional() private readonly cdkScrollable: CdkScrollable | null
+  ) {}
+
+  showExpand(isBottomControl: boolean): boolean {
     return isBottomControl ? this.isGroupExpandable : false;
   }
 
   showPaginationInput(isBottomControl: boolean) {
     return isBottomControl;
   }
 
+  PaginationCallback(pageIndex: number, target: HTMLElement) {
```

Comment:
I think it would be more conventional to start this with a verb. `setPaginationCallback` would be an improvement. Perhaps even better is handlePageChange() since (1) it also emits `pageIndexChanged` and (2) I don't know what you mean by `Callback`.

Reply:
I like handlePageChange.